subjects/printfirsthalf/README.md

```diff
@@ -0,0 +1,46 @@
+## printfirsthalf
+
+### Instructions
+
+Write a function called `PrintFirstHalf()` that takes a `string` as an argument and returns printed the first half of a `string`.
+
+- If the length of the `string` is odd round it down.
+- Prints the first half followed by a newline `\n`.
```

Comment:
This line is unnecessary, these "If's are only for special cases, program already says what to print in a correct output in the instructions.

Reply:
It is fixed.